backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/AggregationEngine.java

```diff
@@ -592,20 +597,18 @@ private LocalDate computeOverrideDueDate(@NonNull final I_C_Invoice_Candidate ic
 					return null;
 				},
 				() -> {
-					logger.debug("Due Date will be set on null for now");
+					logger.debug("Due Date will now be computed based on payment term settings");
 
-					return null;
+					final PaymentTerm paymentTerm = paymentTermRepository.getById(paymentTermId);
+					Timestamp baseLineDate = invoiceCandBL.getBaseLineDate(paymentTerm, ic);
+					if (baseLineDate == null)
+					{
+						baseLineDate = TimeUtil.asTimestamp(computeDateInvoiced(ic));
```

Comment:
consider throwing exception instead of fallback to something else which is not known/documented for the user.